xc/xenops_server_xen.ml

```diff
@@ -848,6 +844,12 @@ module VM = struct
       nomigrate = false;
```

Comment:
why not replacing this whole block with
```ocaml
{ default_persistent_t with 
  VmExtra.build_info = Some build_info
; ty = Some vm.ty
(* Earlier than the PV drivers update time, therefore
    any cached PV driver information will be kept. *)
; last_start_time = 0.0
; profile = profile_of ~vm
}
```
Updating `last_start_time` is superfluous, but I kept it there to keep the comment meaningful